dbt_project.yml

```diff
@@ -26,3 +26,4 @@ on-run-end:
   - "{{ elementary.upload_dbt_artifacts() if not elementary.get_config_var('disable_dbt_artifacts_autoupload') }}"
   - "{{ elementary.upload_run_results(results) if not elementary.get_config_var('disable_run_results') }}"
   - "{{ elementary.handle_tests_results(results) if flags.WHICH in ['test', 'build'] }}"
+  - "{{ elementary.upload_information() }}"
```

Comment:
It seems like we are starting to have lots of on-run-end hooks and I think you mentioned this slows us down because it opens connection to the db. Maybe worth uniting them at this point? If you do so add debug logs so we can monitor their timings?

Reply:
https://github.com/elementary-data/dbt-data-reliability/pull/124 😉